graphene/src/document.rs

```diff
@@ -699,7 +699,10 @@ impl Document {
 			Operation::SetLayerFill { path, color } => {
 				let layer = self.layer_mut(path)?;
 				match &mut layer.data {
-					LayerDataType::Shape(s) => s.style.set_fill(layers::style::Fill::new(*color)),
+					LayerDataType::Shape(s) => {
+						s.style.set_fill(layers::style::Fill::new(*color));
+						s.closed = true;
```

Comment:
I'm surprised this isn't causing the stroke to be drawn from the starting point to the ending point, since I thought that was what the `closed` property meant. However we don't actually want to change the stroke when filling the shape, since the shape is still open and it just happens to have a fill. Perhaps this `closed` property does what we want but it might be deserving of a rename. Pining @otdavies for thoughts.

Reply:
Whether the path is closed is determined by whether or not the `kurbo::BezPath` ends with a `ClosePath` element. This closed field is only used for selection and should probably be called something like `solid` instead.

Reply:
`filled` would probably be clearest, although couldn't we just remove the field entirely and check for whether it has a `Some(...)` or `None` fill color?